app/models/feedback.rb

```diff
@@ -25,7 +25,7 @@ class Feedback < ActiveRecord::Base
   def author_name_and_email
     email_in_brackets = email.present? ? "(#{email})" : "No email provided"
     if author
-      "#{author.name} #{email_in_brackets}"
+      "#{author.name('first_name_with_initial')} #{email_in_brackets}"
```

Comment:
This type of "selecting one of enumerated options" usecase should always use symbols and never plain strings. It documents the case better, makes comparison operations more efficient and uses less memory.